### PR TITLE
Putting back terminal dont-ask in interactive cli

### PR DESCRIPTION
**Pull request by dcheriancisco**
1. removed the un-necessary escaped '-' from YAML regexes
2. addded back terminal dont-ask in YAML which were removed with the grpc-merge

### CODE REVIEW

File: lib/cisco_node_utils/cmd_ref/vpc.yaml [hunk 1]
```diff
@@ -50,16 +50,19 @@ domain:
...
-    get_value: '/^dual\-active exclude interface\-vlan\s+(\S+)/'
+    get_value: '/^dual-active exclude interface-vlan\s+(\S+)/'
     set_value: "<state> dual-active exclude interface-vlan <range>"
   default_value: 'none'
 
 fabricpath_emulated_switch_id: 
   _exclude: [N3k, N5k, N6k, N9k]
   kind: int
+  get_command:  'show running-config vpc all'
+  get_context:  ['/^vpc domain\s*(\d+)$/']
```
Comment (glennmatthews) [inline lib/cisco_node_utils/cmd_ref/vpc.yaml]: The `get_command` and `get_context` appear to be identical to the values provided by the `template`. I understand the need for the custom `set_context` for this property, but why these?

Comment (dcheriancisco) [inline lib/cisco_node_utils/cmd_ref/vpc.yaml]: Not needed, it is my lack of understanding of how template works. I thought if either set_context or get_context is brought to the property level, then the template couldn't be used at all for the property.

File: lib/cisco_node_utils/cmd_ref/vpc.yaml [hunk 2]
```diff
@@ -96,7 +97,9 @@ layer3_peer_routing:
 
 peer_gateway: 
   kind: boolean
-  get_value: '/^peer\-gateway/'
+  get_context:  ['/^vpc domain\s*(\d+)$/']
```
Comment (glennmatthews) [inline lib/cisco_node_utils/cmd_ref/vpc.yaml]: this line isn't needed I think?

**Comment by mikewiebe** [comment]
:+1:

**Comment by dcheriancisco** [comment]
Done with commits. Please Merge

**Pull request merged**